GeeksCoreLibrary/Components/OrderProcess/Services/OrderProcessesService.cs

```diff
@@ -109,7 +113,14 @@ public async Task<OrderProcessSettingsModel> GetOrderProcessSettingsAsync(ulong
                                 IF(clearBasketOnConfirmationPage.value IS NULL OR clearBasketOnConfirmationPage.value = '', '1', clearBasketOnConfirmationPage.value) AS clearBasketOnConfirmationPage,
 	                            CONCAT_WS('', header.value, header.long_value) AS header,
 	                            CONCAT_WS('', footer.value, footer.long_value) AS footer,
-                                CONCAT_WS('', template.value, template.long_value) AS template
+                                CONCAT_WS('', template.value, template.long_value) AS template,
+                                IF(measurementProtocolActive.`value` = 1, TRUE, FALSE) AS measurementProtocolActive,
+                                measurementProtocolItemJson.`value` AS measurementProtocolItemJson,
```

Comment:
Hier graag zo'n `CONCAT_WS` gebruiken met `value` en `long_value`, net zoals bij header/footer/template. Dat is nodig omdat Wiser alle waardes die kleiner zijn dan 1000 tekens opslaat in value en groter dan 1000 tekens in long_value. Als je nu een grotere JSON krijgt, dan werkt het niet meer omdat je nu de `long_value` nooit ophaalt. Hetzelfde geldt voor de andere JSON properties hieronder.